### ISSUE TITLE
FR for also showing expression data to solve a possibly Bitwig specific problem.

### ISSUE THREAD

**Issue by TjShredder** [gbevin/ShowMIDI#44]
In Bitwig I can't get pitch bend to be sent to a MPE Synth from a PlugData patch. I wanted to analyze it with ShowMIDI, as I suspect there is a confusion between Expressions and Midi. I would like to see also Expressions in ShowMIDI.
I need to create an informed bug report and need to know what is happening with Expressions and how or if there is a translation between Midi and expressions. Especially when CLAP is involved. On the VST3 version I can tell it to receive Midi by switching on MPE, on the CLAP version I can't...

**Comment by gbevin** [comment]
What do you mean by expressions?

**Comment by TjShredder** [comment]
Isn't the pitch bend an expression within Bitwig? So the Midi pitch bend is translated into the correct pitch according to the note and the pitch bend range? 
For example I tested my PlugData (CLAP) patch and it does output pitch bend as expected, but it isn't recognized by Equator set to listen to MPE. This might be a Bitwig bug or design quirk...
Also, when I place ShowMIDI (CLAP) before Equator, it will not show any PB Data and in addition it will block the pitch bend Data when played with a MPE controller. ShowMIDI as VST3 does show the incoming pitch bend when set to MPE, but also does not pass it on in a way that Equator reacts to it. The CLAP is supposed to work without the need to set it. But it does not show the changing pitch, only the held note...
As far as I know Steinberg introduced expressions as part of their VST3 specs (not that I ever read them; - )
But exept for pitch bend I got it never working with timbre or aftertouch in Bitwig. This might be again a Bitwig bug/quirk. I just want to send an informed bug report and I hoped that ShowMIDI could deliver all the information I would need...
